README.md

```diff
@@ -4,12 +4,16 @@ This module is used to add share tools to VJ content.
 
 ##Example use
 
+###Requirements
+Sharetools has a jquery dependency. Make sure 'jquery' is findable in your RequireJS paths.
```

Comment:
I didn't realise this was used by IDT. This PR is for changes to the developer-scaffold, where jQuery will be called `jquery`. Could IDT not change the RequireJS path so that it always says `jquery`?